helpers/ExhibitPageFunctions.php

```diff
@@ -92,18 +92,18 @@ function exhibit_builder_page_nav($exhibitPage = null)
     $levelNumber = 1;
     
     foreach ($pagesTrail as $page) {
-        $linkText = $page->title;
+        $linkText = $page->menu_title;
```

Comment:
This is a problem but it looks like `$linkText` is totally unused anyway so it can just be deleted.